bindings.h

```diff
@@ -1 +1,2 @@
+#include "sdkconfig.h"
```

Comment:
appears like this needs to be higher than the idf-component header (`bme280.h`) to compile properly